docs/build-on-linea/run-a-node/use-docker.mdx

```diff
@@ -0,0 +1,151 @@
+---
+description: Use Docker to run a Besu or Geth client as a Linea node.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Use Docker
+
+You can use Docker to run [Hyperledger Besu](https://besu.hyperledger.org/) or [Geth](https://geth.ethereum.org/) as
+a Linea node on the mainnet or testnet. 
+
+:::warning Important
+
+The Besu Docker image does not run on Windows.
+
+:::
+
+## Prerequisites
+
+Download and install [Docker](https://www.docker.com/products/docker-desktop/).
+
+## Run a Besu node
+
+### Step 1. Download configuration files
+
+Download the configuration files for the relevant network (in ZIP format). The configuration files include the network genesis file,
+Docker Compose file and Besu configuration file.
+
+<Tabs groupId="networks" className="my-tabs">
+  <TabItem value="mainnet" label="Mainnet">
+  
+  Download the mainnet [`besu-mainnet` ZIP file](pathname:///files/besu/besu-mainnet.zip).
+  
+  </TabItem>
+  <TabItem value="testnet" label="Testnet">
+  
+  Download the testnet [`besu-goerli` ZIP file](pathname:///files/besu/besu-goerli.zip).
+  
+  </TabItem>
+</Tabs>
+
+### Step 2. Update the Docker Compose file
+
+In the `docker-compose.yaml` file, edit the volumes section by changing `/path/to/your/directory` to the
+directory where you are storing the configuration files. For example:
+
+<Tabs groupId="networks" className="my-tabs">
+  <TabItem value="mainnet" label="Mainnet">
+  
+  ```yaml
+  volumes:
+    - /Users/myuser/besu-mainnet:/config
+  ```
+  
+  </TabItem>
+  <TabItem value="testnet" label="Testnet">
+  
+  ```yaml
+  volumes:
+    - /Users/myuser/besu-goerli:/config
+  ```
+  
+  </TabItem>
+</Tabs>
+
+The folder where you are storing the files will be mounted to the `/config` folder inside the Docker container.
+
+### Step 3. Update the client configuration file
+
+Update the Besu configuration file (`config-snap-[NETWORK].toml`) with your IP public address. To do this, set
```

Comment:
@bgravenorst if you update the zips I think this is all you need to change to `docker-compose.yml`